webapp-django/crashstats/settings/base.py

```diff
@@ -98,6 +98,8 @@ def path(*dirs):
     'waffle',
     'pinax.eventlog',
     'django_jinja',
+
+    'django.contrib.admin',
```

Comment:
I put this last so that when we override the index template in `crashstats`, we'll get our version.

Reply:
I'll add a comment about that in the next PR when I add the overridden index template.

Reply:
I originally had that, but had problems with getting it to work right. I'll try again--maybe I  missed something.

Reply:
Where "I originally had that" is more like "I tried that when I was doing this with Standups, but hit problems". I'll try the `AdminSite` thing in the next PR, but will leave this as is for now.